pom.xml

```diff
@@ -1096,7 +1096,7 @@
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
-        <version>3.9.0</version>
+        <version>3.12.4</version>
```

Comment:
This version was needed in order to run tests in Java 17 and 20. Version 4.x is the one that officially support modern Java versions, but they removed deprecated apis there, so just in case we may be using them, I preferred to leave it with a 3.x version that seem to work.